holoviews/plotting/comms.py

```diff
@@ -57,12 +57,11 @@ class Comm(object):
 
     js_template = ''
 
-    def __init__(self, plot, id=None, on_msg=None):
+    def __init__(self, id=None, on_msg=None):
```

Comment:
Nice to see that all these ``plot`` arguments and the ``_plot`` attribute were unnecessary!

Reply:
Agreed, this has annoyed me in the past but I didn't think to look it was actually used.